planner/utils.py

```diff
@@ -6,7 +6,7 @@
 import time
 import json
 import pytz
-
+import pickle
```

Comment:
can you add this to the requirements.txt please

Reply:
It's a built in library. Should be good to go without a pip install.